### ISSUE TITLE
Punchlist of community-preparedness needs

### ISSUE THREAD

**Issue by quaid** [urbantech/musacapital#14]
@urbantech asked me to take a look at this project and create a gap list of the aspects it needs for lowering the barriers of participation and overall being a welcoming project. For simplicity sake, I'm going to enumerate them here in this single issue using a checklist format, rather than create individual issues for each item. However, it is likely one or more of the items should be broken out into a stand alone issue for working through, so we'll make a new issue for it at that time.

Some of these are items that MUST be done before the Summer of Code AI Hack House and Externship Program starts, some are SHOULD be done items, and the remainders MAY be done as part of the summer program.

Here is the single list, which will grow through editing this description as I look further:

## MUST
- [ ] CONTRIBUTING.md
- [ ] Code of Conduct
- [ ] Stand alone website (even github.io)
- [ ] Stand alone project name
- [ ] GitHub organization for managing the project with more than one owner
- [ ] Real-time Openly archived communication channel (chat e.g. app.element.io)
- [ ] Async Openly archived communication channel (web forum, mailing list)

## SHOULD
- [ ] Primary CTA on the website
- [ ] Project purpose, how to get and use the software, etc. on the branded site
- [ ] Inclusivity proof and statement
- [ ] Release management plan
- [ ] Development and release cycle or model

## MAY
- [ ] Developer documentation quality?
- [ ] User documentation
- [ ] What outreach is done? Social media, blogs, events, etc.
